Cargo.toml

```diff
@@ -22,6 +22,7 @@ jobserver = { version = "0.1.16", optional = true }
 
 [features]
 parallel = ["jobserver"]
+cudart = []
```

Comment:
Oh one thought I just had, now that this is no longer controlling an optional dependency it's probably best to have this be a configuration method on `Build` rather than a compile-time feature

Reply:
I don't quite follow the "on `Build`" thing. Does it refer to `[build-dependecies]`?

Reply:
For reference. Original rationale behind making `cudart` a feature was to ensure that nothing gets broken. I mean if somebody is using `cc-rs` for CUDA **now**, they ought to have some kind of solution for the linking problem. And they wouldn't directly appreciate the interference if `cc-rs` just started linking cudart. In other words it's only appropriate to make it optional and off by default. And I figured that a feature would be least invasive. But it's naturally not the only option. For example, we could instead add `.cudart()` **method**. And not as complementary, but as replacement for `.cuda()`. I mean application would have a choice to use `.cuda()` or `.cudart()`, and not `.cuda()` **and** `.cudart()`.

Reply:
> I don't quite follow the "on `Build`" thing.

Ah! I think I got it now. And the above remark about `.cudart()` is effectively a reply to the suggestions. I suppose remaining question is if `.cudart()` would be considered suitable. Or maybe `.link_with_cudart()` would be considered more appropriate?

Reply:
A method that starts with "cuda" can be considered preferred, because it would appear next to `.cuda()` method in documentation. There already is similar precedent, `.cpp_link_stdlib()`. So, `.cudart()` or `.cuda_link_cudart()`? Or maybe `.cuda_link_rt()`?

Reply:
I've decided to go for `.cudart()`, because corresponding nvcc command-line option is called `--cudart`. And to mimick it even closer it accepts same options `{none|shared|static}`.

Reply:
Ah sorry, but what you ended up on is what I was thinking!